pantheon-bundle/frontend/src/app/searchFilter.tsx

```diff
@@ -57,7 +57,7 @@ class SearchFilter extends Component<any, any> {
       { value: 'Uploaded date', label: 'Uploaded date', disabled: false },
       { value: 'Title', label: 'Title', disabled: false },
       { value: 'Updated date', label: 'Updated date', disabled: false },
-      { value: 'Module type', label: 'Module type', disabled: false }
+      { value: 'Module type', label: 'Content type', disabled: false }
```

Comment:
Can we also consider changing other module instances changes within this PR?

Reply:
changing the key value breaks the sort. We are going to re-do the search page in the near future. For this ticket, the goal is to just update the label.